Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -0,0 +1,442 @@
+# Migrating from Analytics mobile extension to Edge Network using the EdgeBridge extension
+## Overview
+This hands-on tutorial provides end-to-end instructions on how to migrate to Edge Network from Analytics to EdgeBridge mobile extension.
+
+There are other beginning states the customer can be in like using ACP Analytics, v4 extension, etc. These usages can potentially be migrated to AEP and from there, this tutorial can be applied afterwards. However, this should be determined on a case-by-case basis, as for example in the v4 extension case, it may be more effective to simply implement AEP extensions itself without the need for Edge Bridge migration path.
+
+Tutorial setup:
+
+Initial test app that is based on prerequisites (implements Analytics)
+Final test app after all the tutorial steps have been implemented (implements EdgeBridge)
+
```

Comment:
It is a lot to digest in this doc.  Is it possible to have a simple workflow diagram to show overview steps.  It is easier for user to grasp what they are going to do in the tutorial.

Start with an Analytics ready app --> Add Edge and Edge Bridge extensions (now can send context data to edge) -->  Add Assurance (optional but recommended) --> use json to add mapper config (so data makes sense to Analytics solution) --> final app.